### ISSUE TITLE
One Way frequency table: Functionalities allowing to decide if you want to display NA and summary in the table or not

### ISSUE THREAD

**Issue by IssoufKon** [africanmathsinitiative/R-Instat#6670]
**Is your feature request related to a problem? Please describe.**
The problem is that it is not possible to remove the "NA" and "summary" in the one way frequency table.  Why is it possible to do this with the two or three way frequency tables but not with the one way frequency table? In fact, these functionalities don't exist in the "more options" menu so we can't decide if we want them or not in the One Way frequency table.

**Describe the solution you'd like**
In fact, in "One Way frequencies", when we click on the "options" menu, in "more options" of the table, we think that we should add the following functionality:
•	Show summary
•	Show missing values
•	Decimal places for percentage 

**Describe alternatives you've considered**
As in the Two Way frequencies table, these different functionalities will allow us to decide if we want to display the missing value or not in the table, the summary or not, and also to decide the number of digits after the decimal point that we want.

**Additional context**
Capture of the area where we could have more functionalities,
![1](https://user-images.githubusercontent.com/88491307/129292254-dc4c474b-28b8-4670-8c8a-ef22ee7ad693.GIF)

Example with the case of frequency 2 table
![image](https://user-images.githubusercontent.com/88491307/129292348-2aa35b8f-4a4b-4c1c-993c-f1e50bcd86a0.png)

Capture of the table with the NA and Summary that we want to delete.
![image](https://user-images.githubusercontent.com/88491307/129292463-459b5a89-0c94-4437-b749-b83be5a75541.png)

**Comment by Gioche6** [comment]
@Ivanluv , I am taking over this task.

**Comment by Ivanluv** [comment]
@Gioche6 okay

**Comment by Patowhiz** [comment]
@rdstern, @Gioche6 and I are looking into different ways we could add the suggested options. As you have mentioned in item (d) above, we will leave the other options if we find it's taking longer to get an alternative R package that does the same.

**Comment by Patowhiz** [comment]
@rdstern we have decided to just solve item (a), (b) and (c) for this sprint. In future more research could be done on alternative packages that accommodates all the suggested options or we could write our own custom function that does the same.

**Comment by Gioche6** [comment]
@rdstern , @Patowhiz .On solving item _(c)_, I have noted that the `frq` function does not return a data frame, but the result is of different class, namely `"sjmisc_frq", "list"` . However,  Once we convert the result into a data frame, some summary statistics of the result  are lost, namely: Total, valid , mean and sd. 
see the snip below. 
![Screenshot 2021-09-06 164434](https://user-images.githubusercontent.com/73229979/132227310-7fe85df3-f531-4c72-8b9b-5480bb73a6ef.png)

Can I go ahead and use the function `as.data.frame()` to convert the result into a a data frame, so that it can be saved?

**Comment by rdstern** [comment]
@Gioche6 yes please go ahead.  Those are not consistent with the data frame - as you say - so not important to try to add there.

**Comment by Gioche6** [comment]
@rdstern , I made the changes you requested on this PR.
